packages/react-core/src/components/Tabs/examples/Tabs.md

```diff
@@ -71,10 +72,10 @@ class SimpleTabs extends React.Component {
           <Tab eventKey={2} title={<TabTitleText>Database</TabTitleText>}>
             Database
           </Tab>
-          <Tab eventKey={3} title={<TabTitleText>Disabled</TabTitleText>} isDisabled>
```

Comment:
Should the isDisabled (and the isAriaDisabled prop in the next Tab) be removed? The Tab title still conveys "disabled".